src/cogent3/core/new_alignment.py

```diff
@@ -1156,7 +1212,7 @@ def apply_pssm(
         pseudocount: int = 0,
         names: OptList = None,
         ui=None,
-    ) -> numpy.array:
+    ) -> numpy.array:  # refactor: type - how to type hint ui?
```

Comment:
users never set ui, it's from a decorator concerning progress display. I think there's a bigger refactor prospect here if we move to using rich for progress bars.